cypress/fixtures/objects/moduleReqId/moduleReqId.json

```diff
@@ -2583,6 +2583,98 @@
                     }
                 ]
             }
+        },
+        "HdmiInput_Manage":{
+            "HdmiInput.onConnectionChanged - Positive Scenario: Validate Hdmi1 - connected true": {
+                "req": [
+                    {
+                        "method": {
+                            "id": "HdmiInput 1.1",
+                            "description": "Should not be null"
+                        }
+                    }
+                ]
+            }, 
+            "HdmiInput.onConnectionChanged - Positive Scenario: Validate Hdmi1 - connected false": {
+                "req": [
+                    {
+                        "method": {
+                            "id": "HdmiInput 1.2",
+                            "description": "Should not be null"
+                        }
+                    }
+                ]
+            },
+            "HdmiInput.onSignalChanged - Positive Scenario: Validate Hdmi1 - signal unknown": {
+                "req": [
+                    {
+                        "method": {
+                            "id": "HdmiInput 1.3",
+                            "description": "Should not be null"
+                        }
+                    }
+                ]
+            },
+            "HdmiInput.onSignalChanged - Positive Scenario: Validate Hdmi1 - signal none": {
+                "req": [
+                    {
+                        "method": {
+                            "id": "HdmiInput 1.4",
+                            "description": "Should not be null"
+                        }
+                    }
+                ]
+            },
+            "HdmiInput.onSignalChanged - Positive Scenario: Validate Hdmi1 - signal stable": {
+                "req": [
+                    {
+                        "method": {
+                            "id": "HdmiInput 1.5",
+                            "description": "Should not be null"
+                        }
+                    }
+                ]
+            },
+            "HdmiInput.onSignalChanged - Positive Scenario: Validate Hdmi1 - signal unstable": {
+                "req": [
```

Comment:
any test case which does not have a spec document should not have test case added in modulerequirement id